src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -274,6 +279,115 @@ static bool has_32bit_output(const std::shared_ptr<ngraph::Node>& node) {
+                return cnn2dValidatorPtr->ValidateCnn2D(node->get_friendly_name(),
+                    conv_data.input_height, conv_data.input_width, conv_data.input_channel_count,
+                    conv_data.filter_height, conv_data.filter_width, conv_data.filter_channel_count,
+                    conv_data.filter_stride_height, conv_data.filter_stride_width,
+                    conv_data.filter_dilation_height, conv_data.filter_dilation_width,
+                    OvGnaTypeIntFromBytes(config.gnaPrecision.size()), raise_exception);
+            }
+        }
+        return check_dilation(conv_ie->get_dilations()[0], conv_ie->get_dilations()[1]);
+    } else if ((std::dynamic_pointer_cast<ngraph::op::ScaleShiftIE>(node) != nullptr) ||
+               (std::dynamic_pointer_cast<ngraph::op::PowerIE>(node) != nullptr) ||
+               (std::dynamic_pointer_cast<ngraph::opset9::MatMul>(node) != nullptr)) {
+        return true;
+    } else if (std::dynamic_pointer_cast<ngraph::op::FullyConnected>(node) != nullptr) {
+        size_t output_batch_size = node->get_output_shape(0)[0];
+        if (output_batch_size > 8) {
+            if (raise_exception) {
+                THROW_GNA_EXCEPTION << "topology with layer: " + node->get_friendly_name() + ", type: " + node->get_type_name() +
+                ", and batch size(" + std::to_string(output_batch_size) + ") not supported";
+            }
+            return false;
+        }
+        return true;
+    } else if (is_eltwise_add(node) || is_eltwise_mul(node)) {
+        return true;
+    } else if (is_pooling(node)) {
+        auto max_pool = std::dynamic_pointer_cast<ngraph::opset7::MaxPool>(node);
+        if (!max_pool)
+            return false;
+        auto kernels = max_pool->get_kernel();
+        if (2 == kernels.size() && kernels.at(0) > 1 && kernels.at(1) > 1) {
+            const auto cnn2dValidatorPtr = Cnn2D::AbstractValidator::Create(gnaCompileTarget);
+            if (cnn2dValidatorPtr) {
+                auto strides = max_pool->get_strides();
+                return cnn2dValidatorPtr->ValidatePooling2D(max_pool->get_friendly_name(),
+                    kernels[0], kernels[1], strides[0], strides[1], raise_exception);
+            }
+        }
+        return true;
+    } else if (is_crop_affined(node)) {
+        return true;
+    } else if (is_gna_precision_agnostic(node)) {
+        if ((std::dynamic_pointer_cast<ngraph::opset9::Split>(node) != nullptr) ||
+            (std::dynamic_pointer_cast<ngraph::opset9::VariadicSplit>(node) != nullptr)) {
+            bool is_aligned = false;
+            for (size_t i = 0; i < node->get_output_size(); i++) {
+                is_aligned |= is_aligned_split(node, i);
+            }
+            return is_aligned;
+        }
+        // TODO check concat are aligned when transformation will be moved to ngraph
+        return true;
+    } else if (is_activation(node.get())) {
+        return true;
+    }
+    return false;
```

Comment:
same here:
what if `raise_exception==true` ?

Reply:
Nothing. We still have legacy passes. It is ok to have unsupported layers on this point.

Reply:
Shouldnt we throw exception instead of returning false?

Reply:
> Shouldnt we throw exception instead of returning false?

For QN we should not raise exception. 
For LN exception is raised in Load Network itself, when all unsupported operations are collected to print them all. 
Exception is raised here only for cases, which are covered by tests cases where some specific message is expected due to partially supported operation.